### ISSUE TITLE
Support for overrideCompatibilityChangeParameters

### ISSUE THREAD

**Issue by jkwatson** [melix/japicmp-gradle-plugin#43]
I see this is in the maven plugin, and would love it for the gradle version. 

In particular, I would very much like to be able to say that `METHOD_NEW_DEFAULT` does not count as a binary compatibility change.

Is there any way to do this today with the gradle plugin?

**Comment by simonbasle** [comment]
Same here! I've explored filtering and custom rules for that purpose but it doesn't work. Best I was able to do is modify the level of some `Violation` in the rich report, but that only makes the rich report empty (the task is still considered a failure because the underlying `japicmp` call doesn't know of that change).

what would be needed is more of an `Excludes` configuration like for classes and methods: list categories of changes in the configuration and have the plugin configure `japicmp` to consider these categories as compatible.

Japicmp (the library, not the plugin) has a `japicmp.model.JApiCompatibilityChange` enum, and a `japicmp.cmp.JarArchiveComparatorOptions.OverrideCompatibilityChange` class that can be used in japicmp options to mark a JApiCompatibilityChange as binary/source compatible.

I need to go through some red tape before I can submit a PR so I ended up using a `methodExcludes` instead.
Just seeing this issue, it looks like the feature could be interesting to others... Maybe somebody would like to take this?
